components/valueCard/valueCard.css

```diff
@@ -0,0 +1,51 @@
+.budget_and_expenses {
+    width: 70%;
+    display: flex;
+}
+
+.field_info {
+    margin: 10px;
+}
+
+.budget-form-container {
+    width: 30%;
+}
+
+input {
+    display: block;
```

Comment:
'valueCard' does not have a input. this css should only have style that relevant for 'valueCard' component